server/utils/applications/withdrawalReasons.ts

```diff
@@ -13,10 +13,9 @@ export type NewApplicationToBeSubmittedReasons = Extract<
 export type ApplicationProblemReasons = Extract<WithdrawalReason, (typeof applicationProblemReasons)[number]>
 export type OtherReasons = Extract<WithdrawalReason, (typeof placementNoLongerNeededReasons)[number]>
 
-export const withdrawalReasons: Record<WithdrawalReason, string> = {
+export const withdrawalReasons: Record<Exclude<WithdrawalReason, 'other'>, string> = {
   death: 'The person died',
   other_accommodation_identified: 'Other accommodation has been identified',
-  other: 'The placement is not needed for another reason',
```

Comment:
I don't think this needs excluding here, since excluding `'other'` from `placementNoLongerNeededReasons` should do the job? Leaving it in here and then taking the value from here in `withdrawalRadioOptions` might be more transparent?

Reply:
correct